gpii/node_modules/testing/src/CloudBasedOAuth2.js

```diff
@@ -192,12 +197,29 @@ gpii.test.cloudBased.oauth2.sendAccessTokenRequest = function (accessTokenReques
         client_secret: options.client_secret
     };
 
-    formBody = gpii.test.cloudBased.oauth2.filter(formBody, options, "accessTokenForm");
+    gpii.test.cloudBased.oauth2.sendRequest(accessTokenRequest, options, formBody, "accessTokenForm");
+};
+
+gpii.test.cloudBased.oauth2.sendAccessTokenRequestInClientCredentials = function (accessTokenRequest, options) {
+    var formBody = {
+        grant_type: "client_credentials",
+        scope: options.scope,
+        client_id: options.client_id,
+        client_secret: options.client_secret
+    };
 
-    accessTokenRequest.send(gpii.test.stringifyFormBody(formBody));
+    gpii.test.cloudBased.oauth2.sendRequest(accessTokenRequest, options, formBody, "accessTokenForm");
 };
 
-gpii.test.cloudBased.oauth2.verifyAccessTokenResponse = function (body, accessTokenRequest) {
+gpii.test.cloudBased.oauth2.verifyAccessTokenInAuthCodeResponse = function (body, accessTokenRequest) {
```

Comment:
I think that gpii.test.cloudBased.oauth2.verifyAccessTokenInAuthCodeResponse and gpii.test.cloudBased.oauth2.verifyAccessTokenInClientCredentialsResponse are exactly the same?